### PR TITLE
Translate help message in French

### PR DESCRIPTION
**Pull request by Paragoumba**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -238,6 +238,381 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
...
+msgid "    --mflags    <flags>    Pass arguments to makepkg"
+msgstr "    --mflags    <flags>    Passe des arguments à makepkg"
+
+#: src/help.rs:38
+msgid "    --pacman    <file>     pacman command to use"
+msgstr "    --pacman    <file>     Commande pacman à utiliser"
+
+#: src/help.rs:39
+msgid "    --git       <file>     git command to use"
+msgstr "    --git       <fichier>    Commande git à utiliser"
```
Comment (Morganamilo) [inline po/fr.po:365]: Spacing broken due to fichier being longer than file. This also appears to be the only place file is translated.

File: po/fr.po [hunk 2]
```diff
@@ -238,6 +238,381 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
...
+#: src/help.rs:79
+msgid ""
+"    --topdown              Shows repository's packages first and then AUR's"
+msgstr ""
+"    --topdown              Affiche les paquets du dépôt et ceux de l'AUR "
+"ensuite"
+
+#: src/help.rs:81
+msgid "show specific options:"
+msgstr "affiche les options spécifiques :"
```
Comment (Morganamilo) [inline po/fr.po:578]: show is lowercase because it's the name of the flag `--show`. Seems as this is changed to a normal word it should be capitalised.

Comment (Morganamilo) [inline po/fr.po:578]: Other translations start with options. What's the difference?

File: po/fr.po [hunk 3]
```diff
@@ -238,6 +238,381 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
...
+
+#: src/help.rs:49
+msgid "    --fmflags   <flags>    Pass arguments to file manager"
+msgstr ""
+"    --fmflags   <flags>    Passe des arguments au gestionnaire de fichier"
+
+#: src/help.rs:51
+#, fuzzy
+msgid "    --completioninterval   <n> Time in days to refresh completion cache"
+msgstr ""
```
Comment (Morganamilo) [inline po/fr.po:413]: Broken.

File: po/fr.po (hunk 2 above)
Comment (Paragoumba) [reply po/fr.po:578]: I didn't understand well what it meant. I fixed it

File: po/fr.po (hunk 3 above)
Comment (Paragoumba) [reply po/fr.po]: This seems to be created by poedit. I removed it manually

**Comment by Paragoumba** [comment]
There seems to be a spacing problem with --completioninterval in English so i didn't fix it in French

>     --completioninterval   <n> Time in days to refresh completion cache
>     --sortby    <field>    Sort AUR results by a specific field during search
>     --searchby  <field>    Search for packages using a specified field

File: po/fr.po [hunk 4]
```diff
@@ -238,6 +238,354 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
...
+#: src/help.rs:48
+msgid "    --fm        <file>     File manager to use for PKGBUILD review"
+msgstr "    --fm        <fichier>      Gestionnaire de fichier à utiliser pour la relecture des PKGBUILDs"
+
+#: src/help.rs:49
+msgid "    --fmflags   <flags>    Pass arguments to file manager"
+msgstr "    --fmflags   <arguments>    Passe des arguments au gestionnaire de fichier"
+
+#: src/help.rs:51
+msgid "    --completioninterval   <n> Time in days to refresh completion cache"
```
Comment (Paragoumba) [inline po/fr.po:407]: There seems to be a spacing problem with --completioninterval in English so i didn't fix it in French

>     --completioninterval   <n> Time in days to refresh completion cache
>     --sortby    <field>    Sort AUR results by a specific field during search
>     --searchby  <field>    Search for packages using a specified field

Comment (Morganamilo) [reply po/fr.po:407]: It was just done that way to avoid pushing everything to the right more.

File: po/fr.po [hunk 5]
```diff
@@ -238,6 +238,354 @@ msgstr "Pacman est en cours d'utilisation, veuillez patienter..."
...
+"    --clonedir  <dir>      Dossier utilisé pour télécharger et exécuter "
+"PKGBUILDs"
+
+#: src/help.rs:36
+msgid "    --makepkg   <file>     makepkg command to use"
+msgstr "    --makepkg   <fichier>      Commande makepkg à utiliser"
+
+#: src/help.rs:37
+msgid "    --mflags    <flags>    Pass arguments to makepkg"
+msgstr "    --mflags    <arguments>    Passe des arguments à makepkg"
```
Comment (Morganamilo) [inline po/fr.po:356]: Could you shorten this to <args> so the text after can line up?

Comment (Morganamilo) [reply po/fr.po:356]: Specifically <arguments> to <args>

Comment (Paragoumba) [reply po/fr.po:356]: I checked and didn't find a shorter word. args is English not French, it will be understandable but a bit weird. If you really want it to be aligned I can change it

Comment (Morganamilo) [reply po/fr.po:356]: Okay leave it as is then. But I don't really get why the same word can't be shortened.
